src/test/java/com/wolox/training/controllers/BookControllerTest.java

```diff
@@ -0,0 +1,148 @@
+package com.wolox.training.controllers;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+
+import com.wolox.training.controllers.vo.BookVO;
+import com.wolox.training.models.Book;
+import com.wolox.training.services.BookService;
+import java.util.Arrays;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+@WebMvcTest(BookController.class)
+public class BookControllerTest {
+
+    private final static String URL = "/api/v1/books";
+
+    @Autowired
+    private MockMvc mvc;
+
+    @MockBean
+    private BookService mockBookService;
+
+    private Book oneTestBook;
+
+    @BeforeEach
+    public void setUp() {
+        oneTestBook = new Book("Terror", "Stephen King", "img.png", "It", "Worst clown ever", "Viking Press", "1986",
+                1502, "4578-8665");
+    }
+
+    @Test
+    public void whenCreateBook_thenReturnCreatedBook() throws Exception {
+        Mockito.when(mockBookService.create(any(BookVO.class))).thenReturn(oneTestBook);
+        mvc.perform(MockMvcRequestBuilders.post(URL)
+                .content("{}")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(MockMvcResultMatchers.status().isCreated())
+                .andExpect(MockMvcResultMatchers.content().json("{\n"
+                        + "    \"id\": 0,\n"
```

Comment:
Hi Joel, we used to put this JSONs in a file, so you can re-use it. But for me, now you can just replace it with a Constant.